src/common/state/table.h

```diff
@@ -44,6 +44,11 @@ typedef struct {
   table_fail_callback fail_callback;
 } RetryInfo;
 
+static const RetryInfo heartbeat_retry = {
```

Comment:
I think it's valid?

Yeah, it's a bit of a hack, but I thought it was better than having to allocate/deallocate the memory every time. Probably we should rethink this one we do redis.c redux. :)